Makefile

```diff
@@ -1,18 +1,28 @@
-PROJECT_NAME := cpclermont/lighthouse-ci-action
-VERSION := 1.0.0
-GITSHA:= $(shell echo $$(git describe --always --long --dirty))
+USER_NAME := poitrin
+PROJECT_NAME := $(USER_NAME)/lighthouse-ci-action
+VERSION := 1.1.0
+GITSHA := $(shell echo $$(git describe --always --long --dirty))
+DOCKER_COMMAND := podman
+PACKAGE_REGISTRY_URL := ghcr.io
 
 export GITSHA
 export VERSION
 
+# Make sure to create an access token and save it under `CR_PAT` before running the login command. Instructions:
+# https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token
+# You will need scopes `read:packages`, `write:packages`, `delete:packages`
+
 base: Dockerfile
-	DOCKER_BUILDKIT=1 docker build -t $(PROJECT_NAME):$(VERSION) -t $(PROJECT_NAME):$(GITSHA) - < Dockerfile.base
+	DOCKER_BUILDKIT=1 $(DOCKER_COMMAND) build --platform linux/amd64 -t $(PROJECT_NAME):$(VERSION) -t $(PROJECT_NAME):$(GITSHA) - < Dockerfile.base
```

Comment:
`--platform linux/amd64` to avoid error `The requested image's platform (linux/arm64/v8) does not match the detected host platform (linux/amd64) and no specific platform was requested`

(I initially built the image on an M1 Mac, whereas CI runs with "conventional" architectures.)